lib/crypto.js

```diff
@@ -0,0 +1,28 @@
+// browser crypto, used in browserify instead of shimming node's crypto package
+
+(function(global, module, define) {
+
+
+function randomBytes(width) {
+  var out = new Uint8Array(width);
+  (global.crypto || global.msCrypto).getRandomValues(out);
```

Comment:
I"m not sure, but we check for the un-prefixed version first. This is probably a safe fallback.